contracts/Pool.sol

```diff
@@ -80,6 +81,11 @@ contract Pool {
         mapping(uint64 => int256) amtDeltas;
     }
 
+    struct ReceiverWeight {
+        address receiver;
+        uint32 weight;
+    }
```

Comment:
Suggestions:
```suggestion
    struct Receiver {
        address address;
        uint32 weight;
    }
```

Reply:
Good point, but `struct Receiver` is already defined on line 73. It's used to store the complete information about a receiver and I think that it fits this name better than a container for an address-weight pair.